bip-0001.mediawiki

```diff
@@ -67,7 +67,7 @@ Each BIP should have the following parts:
 
 * Abstract -- a short (~200 word) description of the technical issue being addressed.
 
-* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example) or licensed under the Open Publication License.
+* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example).
```

Comment:
@laanwj We did the latter several months ago, but it got held up by arguments over what wiki to use for the (unrelated) BIP Comments. Might as well just finish it...